src/Application.js

```diff
@@ -167,6 +169,10 @@ const __class = declare('crm.Application', [Application], {
       }
     } catch (e) {}// eslint-disable-line
   },
+  _setupToasts: function _setupToasts() {
+    this.Toast = new Toast();
```

Comment:
Can we make this lower case? Generally we should only capitalize constructor functions, not the instance (with a few global exceptions).